### ISSUE TITLE
Manage Stake Modal flow - FE/BE Integration

### ISSUE THREAD

**Issue by masvelio** [hicommonwealth/commonwealth#6330]
## Description

In #5476 includes UI work. To split this into smaller piece of work, we need to integrate FE with BE separately.
More info about behaviours can be found in #5476 description

**Comment by masvelio** [comment]
blocked by #5476

**Comment by HIM92** [comment]
New acceptance criteria: 
1) When a user is minting stake, and they are signed in with one address on a community that does not have ETH, the user address with the live session key is the default address for the minting transaction. The user has to select the change of address to one that is funded with ETH. It is not automatic.

2) We are changing the terms of acquiring and reducing your Community Stake from "buying" and "selling" to "mint" and "burn." We are also changing any mention of purchase, sale and other words that imply buying and selling. I've annotated the designs here as bubble comments. The UX areas that are affected are the stake banner, stake modal, stake button on the side bar, as well as the transaction complete event on the modal: 

https://www.figma.com/file/KOEVNGXStBhmX3NnYq7MxP/Community-Stake?type=design&node-id=1117-201363&mode=design&t=Dkl0Zq27TEgEG7nF-0

**Comment by HIM92** [comment]
Copy for vote weight explainer: 

"Your vote weight is based on membership and the amount of Stake you have in your wallet. Each member of this community gets 1 vote for joining. All other vote weight is provided by the amount of stake in your wallet, and the vote weight provided per stake by the community's stake contract."

**Comment by HIM92** [comment]
**Change in ETH decimal behavior**

1. On the vote weigh modal, reduce the number of maximum decimals we show for ETH figures to 8. So the smallest viewable integer of ETH we show is 0.00000001
2. On the vote weight side bar, reduce the number of maximum decimals we show for ETH figures to 8. So the smallest viewable integer of ETH we show is 0.00000001
